QUARKUS-3461.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-3461 Verify behaviour and functionality of Quarkus CLI update command with RHBQ
+- JIRA link: https://issues.redhat.com/browse/QUARKUS-3461
+- Quarkus Guides:
+    - https://quarkus.io/guides/update-quarkus
+
+## Scope of the testing
+- Exploratory testing of the feature over Quarkus upstream and last released version of RHBQ
+- Verify that the `quarkus update` command explicitly warns users about deprecated/unsupported extensions during migration to the newer version.
+  In most cases deprecated extensions are moved to `io.quarkiverse`, which is accepted for Quarkus upstream but not for RHBQ.
+- Test coverage in QQE TS:
+  - Create a Quarkus/RHBQ app with the oldest possible stream and update to the latest version - verify new Quarkus platform version and app functionality
```

Comment:
This doesn't tell me what are you going to test it because `quarkus update` will only have effect if some of recipes are applicable. If you meant to prepare the old version of app in a way that it needs to be updated, please be more specific in the plan.